src/content/zh-tw/_footer.yaml

```diff
@@ -0,0 +1,55 @@
+---
+footer:
+- promos:
+  - description: Chromium博客上的最新消息。
+    icon: https://www.gstatic.com/images/icons/material/product/2x/chrome_chromium_64dp.png
+    label: Chromium博客
+    path: https://blog.chromium.org
+  - description: 復刻我們的代碼示例和其他開源項目。
+    icon: "/site-assets/logo-github.svg"
+    label: GitHub
+    path: https://github.com/googlechrome
+  - description: 在Twitter上與@ChromiumDev聯繫。
+    label: 推特
+    icon: "/site-assets/logo-twitter.svg"
+    path: https://twitter.com/chromiumdev
+  - description: 看看我們的視頻。
+    icon: https://www.gstatic.com/images/icons/material/product/2x/youtube_64dp.png
+    label: 視頻
+    path: "/web/shows/"
+  - description: 參加開發者活動。
+    icon_name: 活動
+    label: 活動
+    path: "/web/events/"
+- linkboxes:
+  - contents:
+    - label: 建立給億萬用戶
+      path: "/web/billions/"
+    - label: Chrome擴展程式文件
+      path: https://developers.chrome.com/extensions/api_index
+    - label: 開發者反饋計劃
+      path: "/web/feedback/"
+    - label: Progressive Web Apps
+      path: "/web/progressive-web-apps/"
+    name: 關鍵主題
+    icon_name: star
+  - contents:
+    - label: 教練帶領培訓
+      path: "/web/ilt/"
+    - label: PWA培訓機構
+      path: "/web/events/pwa-agency-training/"
+    - label: 在Udacity的網絡課程
+      path: "/web/shows/udacity/"
+    icon_name: 學校
```

Comment:
icon_names shouldn't be translated

Reply:
@kennith Could you modify it?

Reply:
@sotayamashita Should I change in GitHub or GitLocalize?

Reply:
@kennith It's kind of awkward but send a PR to this repository: https://github.com/agektmr/WebFundamentals/tree/gitlocalize-328

Reply:
@agektmr done. Please review.